.github/workflows/sphinx-build.yml

```diff
@@ -15,14 +15,9 @@ jobs:
 
     - uses: actions/checkout@v2
 
-    # note (crcrpar): `readthedocs/build:latest` seems to use Python3.8.
-    # Optuna uses that image in CircleCI documentation build,
-    # so we can use Python3.7 here for the backward compatibility
-    # of Pickle's protocol version (Python3.8's pickle's default protocol version
-    # is not compatible completely with Python3.7 and Python3.6).
```

Comment:
[comment] The cache of this workflow can be downloaded via GitHub Actions page, and it is useful for developers to shorten the build time. By changing the Python version from 3.7 to 3.8, I think the developers who use Python 3.7 need to migrate their development environments to Python 3.8 to use the cache. IMO, this is acceptable because (1) they can skip the local build and use doc build CI jobs (2) EOL of Python 3.7 is approaching (1.5 years), and (3) they can build the docs without the cache.